tests/plugins/remove_duplicates.test.ts

```diff
@@ -0,0 +1,242 @@
+import { CommandResult } from "../../src";
+import { toZone } from "../../src/helpers";
+import { getCell, getEvaluatedCell } from "../test_helpers";
+import { merge, selectCell, setFormat, setSelection } from "../test_helpers/commands_helpers";
+import {
+  createModelFromGrid,
+  getRangeFormatsAsMatrix,
+  getRangeValuesAsMatrix,
+} from "../test_helpers/helpers";
+
+// prettier-ignore
+const grid = {
```

Comment:
same here, but worst :sweat_smile: 
I need to have *24* cells in mind. I bet most test scenario usually only need a handful of cells (max 4).